### PR TITLE
Use standard version of kafka broker

### PR DESCRIPTION
**Pull request by mark-r-m**
Use standard version of Kafka broker
Add broker shell command
Standardise some test var names
Add readme samples

### CODE REVIEW

**Review by mikew94 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Kafka2Hbase
+# Kafka2
```
Comment (dan-hill2802) [inline README.md:1]: looks like a mistake

File: README.md [hunk 2]
```diff
@@ -356,3 +357,49 @@ defaulted in the `entrypoint.sh` script.
...
+   ls
+   ```
+
+Check the current list of topics
+   ```shell script
+   ./kafka-topics.sh --zookeeper zookeeper:2181 --list
+   ```
+
+Make a new topic
+...note that doing it this way we must specify the partitions, while through code it is defaulted at the server level.
```
Comment (dan-hill2802) [inline README.md:381]: this renders a bit funny, did you want multiple lines?

File: README.md (hunk 1 above)
Comment (mark-r-m) [reply README.md]: Yes, thx!

File: README.md (hunk 2 above)
Comment (mark-r-m) [reply README.md:384]: Ahh yes, thx

**Review by DanielChicot — APPROVED** [review]

**Pull request merged**